multinode_integration_tests/tests/bookkeeping_test.rs

```diff
@@ -112,6 +112,7 @@ pub fn start_real_node(cluster: &mut MASQNodeCluster, neighbor: NodeReference) -
     let index = cluster.next_index();
     cluster.start_real_node(
         NodeStartupConfigBuilder::standard()
+            .db_password(None)
```

Comment:
...so that the recall-the-old-public-key machinery will not groan into action for this test, which doesn't require it.